packages/@aws-cdk/aws-backup/lib/vault.ts

```diff
@@ -0,0 +1,159 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as kms from '@aws-cdk/aws-kms';
+import * as sns from '@aws-cdk/aws-sns';
+import { Aws, Construct, IResource, RemovalPolicy, Resource } from '@aws-cdk/core';
+import { CfnBackupVault } from './backup.generated';
+
+/**
+ * A backup vault
+ */
+export interface IBackupVault extends IResource {
+  /**
+   * The name of a logical container where backups are stored.
+   *
+   * @attribute
+   */
+  readonly backupVaultName: string;
+}
+
+/**
+ * Properties for a BackupVault
+ */
+export interface BackupVaultProps {
+  /**
+   * The name of a logical container where backups are stored.
+   *
+   * @default - A CDK generated name
+   */
+  readonly backupVaultName?: string;
+
+  /**
+   * A resource-based policy that is used to manage access permissions on the
+   * backup vault.
+   *
+   * @default - access is not restricted
+   */
+  readonly accessPolicy?: iam.PolicyDocument;
+
+  /**
+   * The server-side encryption key to use to protect your backups.
+   *
+   * @default - an Amazon managed KMS key
+   */
+  readonly encryptionKey?: kms.IKey;
+
+  /**
+   * A SNS topic to send vault events to.
+   *
+   * @default - no notifications
+   */
+  readonly notificationTopic?: sns.ITopic
+
+  /**
+   * The vault events to send.
+   *
+   * @default - all vault events if `notificationTopic` is defined
+   */
+  readonly notificationEvents?: BackupVaultEvents[];
+
+  /**
+   * The removal policy to apply to the vault.
+   *
+   * @default RemovalPolicy.RETAIN
+   */
+  readonly removalPolicy?: RemovalPolicy;
+}
+
+/**
+ * Backup vault events
```

Comment:
Maybe we want to point to the docs?
`@see https://docs.aws.amazon.com/aws-backup/latest/devguide/sns-notifications.html`